python/tox.ini

```diff
@@ -0,0 +1,56 @@
+[tox]
+envlist = py{27,36}, coverage
+skip_missing_interpreters = true
+
+
+[testenv]
+passenv = APPVEYOR* CI SSL_CERT_FILE TEST_MARKER TRAVIS*
+setenv = COVERAGE_FILE={toxworkdir}/.coverage.{envname}
+usedevelop = True
+deps = -rrequirements.txt
+       -rtest/requirements.txt
+# TODO: way too many warnings in tests, remove `-p no:warnings` and get rid of them all
+commands = pytest {posargs:-vv -p no:warnings -m {env:TEST_MARKER:main} --doctest-modules --cov-report term-missing --cov=src/}
+#          python -m unittest discover -s test/ -p '*_test.py'
```

Comment:
One can simply comment the `pytest` line here, and uncomment the `unittest` one to switch to unittest to exercise the tests